src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -718,6 +718,9 @@
   <data name="CloseOnExitInfoBar.Message" xml:space="preserve">
     <value>Termination behavior can be configured in advanced profile settings.</value>
   </data>
+  <data name="SetAsDefaultInfoBar.Message" xml:space="preserve">
+    <value>Windows Terminal can be set as your default terminal application in your settings.</value>
+  </data>

```

Comment:
(Offline discussion w/ @cinnamon-msft ) "Looks good"

Reply:
I changed it to
> Windows Terminal can be set as <ins>the</ins> default terminal application in your settings.